spec/at-rules/for.md

```diff
@@ -22,38 +22,38 @@
 
 To execute a `@for` rule `rule`:
 
-* Let `from` be the result of evaluating the expression in `FromDeclaration`
-
-* Assert that `from` is a number
```

Comment:
I validated the number here explicitly to match the order of error reporting in implementations (even though I wrote it the wrong way)

Reply:
If a stylesheet is going to produce an error, implementations have a lot of freedom in which specific error to present to the user and how to present it. In this case, if `from` isn't a number *and* `ToDeclaration` has an evaluation error, implementations are free to surface either error first and thus to check for errors in either order. This flexibility of implementation helps keep the spec simple without ruling out optimizations or cosmetic differences in implementations.